exercises/L-functions-nested/exercise.js

```diff
@@ -3,3 +3,34 @@ var mentor2 = "Irina";
 var mentor3 = "Mimi";
 var mentor4 = "Rob";
 var mentor5 = "Yohannes";
+
+//THIS SHOULD WORK!
+function createGreeting(name) {
+  var greeting = `Helloooo ${name}`;
+  return greeting.toUpperCase();
+}
```

Comment:
the idea of this exercise was to have 2 separate functions, one that returns the name in uppercase and another that returns the greeting by calling the first function to get the name